ftl/python/builder.py

```diff
@@ -113,6 +113,13 @@ def Build(self):
             exposed_ports=self._args.exposed_ports)
         app.BuildLayer()
         lyr_imgs.append(app.GetImage())
+        additional_directory = base_builder.AppLayerBuilder(
```

Comment:
Right now this flag is available for all runtimes but only used in python, to fix this it would make sense to either:
only add this to python and change the flag to work in python
see: https://github.com/GoogleCloudPlatform/runtimes-common/blob/master/ftl/common/args.py#L136
OR
add this snippet and an identical test to each runtime

Reply:
Added to node and php as well.